shell/edit/workload/index.vue

```diff
@@ -28,6 +28,40 @@ export default {
       if ( container ) {
         this.selectContainer(container);
       }
+    },
+
+    /**
+     * Find error exceptions to be mapped for each case
+     */
+    mapError(error) {
+      switch (true) {
+      case error.includes('violates PodSecurity'): {
+        const match = error.match(/(?<=\")(.*?)(?=\")/gi);
+        const name = match[0];
+        const policy = match[2];
+
+        return {
+          message: `Pod "${ name }" Security Policy Violation "${ policy }"`,
+          icon:    'icon-pod_security'
+        };
+      }
+
+      default:
+        break;
+      }
+    },
```

Comment:
Is this dependent on https://github.com/rancher/dashboard/pull/7918? I don't see how to create a workload that will result in an error response due to a security policy violation; the most I've seen is a 201 response with an error header, which looks like what the other PR is for. The other changes to error formatting don't appear to cause issues, I just haven't verified that this part works as intended.

Reply:
No these are 2 different things. The error is for restricted, the banner is a growl for warning. In the videos and issue there's the steps to set the PSA to make it fail in the Pod creation.